src/main/java/net/minecraftforge/common/ForgeModContainer.java

```diff
@@ -163,6 +164,11 @@ public ForgeModContainer()
         INSTANCE = this;
     }
 
+    public static List<String> getUnificationPriorityModIds()
+    {
+        return unmodifiableUnificationPriorityModIds;
```

Comment:
should this be immutable-wrapped?

Reply:
It's a `Collections.unmodifiableList`

Reply:
He probably means that the list should not change in content after access (be a snapshot)

Reply:
It should only change when the config is loaded, so I don't think it's that important either way.